plugins/remoterx_

```diff
@@ -1,67 +1,80 @@
-#!/usr/bin/python
+#!/usr/bin/python2.7
```

Comment:
Always make sure to specify the version. Print in this case, for example is incompatible with python3

Reply:
Roger.